src/main/java/bio/terra/service/tabulardata/azure/StorageTableDao.java

```diff
@@ -0,0 +1,192 @@
+package bio.terra.service.tabulardata.azure;
+
+import bio.terra.model.BulkLoadFileState;
+import bio.terra.model.BulkLoadHistoryModel;
+import bio.terra.service.snapshot.exception.CorruptMetadataException;
+import bio.terra.service.tabulardata.LoadHistoryUtil;
+import com.azure.data.tables.TableClient;
+import com.azure.data.tables.TableServiceClient;
+import com.azure.data.tables.models.ListEntitiesOptions;
+import com.azure.data.tables.models.TableEntity;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class StorageTableDao {
```

Comment:
Has the ship already sailed on what we're calling our azure classes? Because `StorageTable` is not a good name for something, even if it's what azure calls this. Like we have `GcsPdao`, this could be `AzurePdao`.